package-lock.json

```diff
@@ -11,6 +11,7 @@
       "dependencies": {
         "@oclif/command": "^1.5.19",
         "@oclif/config": "^1.13.3",
+        "@oclif/core": "^2.9.4",
```

Comment:
this will cause many problems. I'm working on porting everything to the new version, I think it'll be best to just stick with `@oclif/command` for now. `@oclif/core` is a rewrite essentially, so I think it'll break things to have both.